Dictionaries/nld_OCRFixReplaceList.xml

```diff
@@ -26,5 +22,7 @@
   <BeginLines />
   <EndLines />
   <WholeLines />
-  <RegularExpressions />
+  <RegularExpressions>
+    <RegEx find="\bln(?=\p{Ll}{2})" replaceWith="In" />
```

Comment:
There are well over 2000 Dutch words starting with "in", not including the (countless) combined words where "in" acts as an amplifier, like "ingelukkig" and "inslecht".

The regex pattern `\bln(?=\p{Ll}{2})` is already quite restrictive, *van Dale* lists exactly two words starting with "ln": "lno" and "ln". Hence, one might argue that the whole-word-transformation `<Word from="ln" to="In" />` is unsafe, but, obviously, that would be “spĳkers op laag water zoeken.”  :smile: